inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/eltwise.cpp

```diff
@@ -41,10 +41,17 @@ std::vector<ngraph::helpers::EltwiseTypes> eltwiseOpTypes = {
         ngraph::helpers::EltwiseTypes::ADD
 };
 
-std::map<std::string, std::string> additional_config = {
-        {"GNA_DEVICE_MODE", "GNA_SW_EXACT"},
-        {"GNA_SCALE_FACTOR_0", "1638.4"},
-        {"GNA_SCALE_FACTOR_1", "1638.4"}
+std::vector<std::map<std::string, std::string>> additional_config = {
+        {
+                {"GNA_DEVICE_MODE", "GNA_SW_EXACT"},
```

Comment:
jfyi, it would be better to separately combine different values of the same configuration parameter,  e.g.
{"GNA_SW_FP32",  "GNA_SW_EXACT"} and combine them with test-specific parameters.